cli/cmd/release_create.go

```diff
@@ -349,6 +349,10 @@ func encodeKotsFile(prefix, path string, info os.FileInfo, err error) (*kotstype
 	switch ext {
 	case ".tgz", ".gz", ".yaml", ".yml":
 		// continue
+	case ".css":
+		// continue
+	case ".woff", ".woff2", ".ttf", ".otf", ".eot", ".svg":
+		// continue
 	default:
 		return nil, nil
 	}
```

Comment:
I'd be tempted to take this out into a function, then call `if !supportedExt(ext); return nil, nil` or something. Took me a second to read what was going on here.